jumpscale/packages/vdc_dashboard/frontend/components/License.vue

```diff
@@ -37,12 +37,10 @@
       <v-card>
         <v-card-title class="headline"> Welcome to ThreeFold Now </v-card-title>
         <v-card-text>
-          This marketplace is a showcase of open source peer-to-peer apps built on top of the TF Grid. We are in demo mode
-          and running on testnet. Note your deployment will be cancelled automatically after three hours.
-          Forgive any instability you might encounter while our developers work out the kinks.
+          This marketplace is a showcase of open source peer-to-peer apps built on top of the TF Grid.
```

Comment:
I think it's not not a showcase anymore ?

Reply:
We can remove the whole message i think, this was intended to be a warning for solutions deletion

Reply:
@xmonader what is your opinion?

Reply:
@rkhamis can you please provide a better phrasing?

Reply:
the one @MElborolossy has is quite alright!

Reply:
the marketplace is a platform for opensource peer-to-peer apps built on top of TF grid


what do you think guys?